Formula/spidermonkey.rb

```diff
@@ -1,18 +1,13 @@
 class Spidermonkey < Formula
   desc "JavaScript-C Engine"
-  homepage "https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey"
+  homepage "https://spidermonkey.dev"
   url "https://archive.mozilla.org/pub/mozilla.org/js/js185-1.0.0.tar.gz"
   version "1.8.5"
   sha256 "5d12f7e1f5b4a99436685d97b9b7b75f094d33580227aa998c406bbae6f2a687"
   license "MPL-1.1"
   revision 4
   head "https://hg.mozilla.org/mozilla-central", using: :hg
 
-  livecheck do
```

Comment:
An alternative livecheck block was proposed in #78997.

Reply:
I've updated this to check the Firefox releases page, as that's less likely to be updated until after a release is published. Checking the [Firefox releases directory](https://ftp.mozilla.org/pub/firefox/releases/) technically works but there's a chance that a directory for a new version may be created before the release is published.